openapi/spec.go

```diff
@@ -3,24 +3,24 @@ package openapi
 // OpenAPI represents the root document object of
 // an OpenAPI document.
 type OpenAPI struct {
-	OpenAPI    string      `json:"openapi" yaml:"openapi"`
-	Info       *Info       `json:"info" yaml:"info"`
-	Servers    []*Server   `json:"servers,omitempty" yaml:"servers,omitempty"`
-	Paths      Paths       `json:"paths" yaml:"paths"`
-	Components *Components `json:"components,omitempty" yaml:"components,omitempty"`
-	Tags       []*Tag      `json:"tags,omitempty" yaml:"tags,omitempty"`
-	Security   []*Security `json:"security,omitempty" yaml:"security,omitempty"`
+	OpenAPI    string                 `json:"openapi" yaml:"openapi"`
+	Info       *Info                  `json:"info" yaml:"info"`
+	Servers    []*Server              `json:"servers,omitempty" yaml:"servers,omitempty"`
+	Paths      Paths                  `json:"paths" yaml:"paths"`
+	Components *Components            `json:"components,omitempty" yaml:"components,omitempty"`
+	Tags       []*Tag                 `json:"tags,omitempty" yaml:"tags,omitempty"`
+	Security   map[string]interface{} `json:"security,omitempty" yaml:"security,omitempty"`
```

Comment:
Should be a `Security Object` according to the spec, see https://swagger.io/specification/#openapi-object